core/drivers/clk/sam/sama5d2_clk.c

```diff
@@ -0,0 +1,594 @@
+// SPDX-License-Identifier: GPL-2.0+ or BSD-3-Clause
+/*
+ * Copyright (c) 2021, Microchip
+ */
+#include <assert.h>
+#include <initcall.h>
+#include <kernel/boot.h>
+#include <libfdt.h>
```

Comment:
move below line 10

Reply:
Remove initcall.h entirely since not needed anymore.